### ISSUE TITLE
Sending feedback to GA through GTM

### ISSUE THREAD

**Issue by eeeschwartz** [laurenancona/unified-analytics#4]
The [ga-feedback](https://github.com/luckyshot/ga-feedback) idea on the roadmap is really interesting. If helpful, here's [my writeup](http://erikschwartz.net/2016-01-23-google-analytics-events-in-google-tag-manager/) of how to send events to GA through GTM.

Steal/improve/ignore as needed :)

**Comment by laurenancona** [comment]
I love that you're writing about this. It's almost as if we're coming from opposite ends of the analytics implementation spectrum; the more I'm learning to code I understand so much of what's happening in the context of how I used to essentially construct Javascript via the Tag Manager UI, and it seems as if you're having the inverse experience. Thanks for this, I have some thoughts but want to reread later.

**Comment by eeeschwartz** [comment]
Definitely. And when plugging the ga-feedback widget into a site using GTM, I quickly realized that [their submission approach](https://github.com/luckyshot/ga-feedback/blob/master/feedback.dev.js#L106) needed to be adapted.

GTM's dataLayer object is a nice way to make extra data available to GTM beyond what's attached to the triggering element. In this case, it's a reasonable way to pull a value out of the form being submitted.

**Comment by laurenancona** [comment]
Ah, right. I spec'ed the idea pretty thoroughly in concept, but keep forgetting to do a trial run of implementing. It's the one feature we won't directly be using here at the city, which is why it's been stuck in `Roadmap` limbo for so long. I do have a few new smaller tags to add, like tracking phone call touch events from mobile clients, and some video play tracking.
